### ISSUE TITLE
Create testing suite for CSV

### ISSUE THREAD

**Issue by dsturgell** [retaildevcrews/databricks-scim-automation#86]
## Description:
- Create a testing suite with mocha/chai for src/app (@databricks-scim-automation/csv)

anflinch - Note on things to look at:
- I think there are (at least) two bugs right now:
  - When there is a row with not enough entries, the app does not fail with a clear error
  - When there are multiple rows of SCIM connectors to create, and one of the nth rows is missing an entry, the app does not fail.  
- Basically, we should try to fail if any row in the csv does not have the same number of entries as the template header .
